docs/_blog/2025_q1_update.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "2025 Q1 Update: What Have We Been Up To?"
+description: "2025 Q1 Update on the Bitcoin Dev Kit Project"
+authors:
+    - thunderbiscuit
+date: "2025-04-01"
+tags: ["BDK", "project"]
+draft: false
+---
+
+The bitcoindevkit team is hitting cruising speed after the release of the 1.0 bdk_wallet API. We are implementing a new release cadence of 8-week cycles, planning to do a feature release (1.1, 1.2, etc.) on [these dates for 2025](
+https://github.com/bitcoindevkit/bdk/milestones).
+
+Here are some of the notable releases and changes over Q1 to the software libraries we maintain:
+- **The new bdk_wallet 1.1 library is out.** This one includes some of the goodies we could not release in 1.0, including support for Testnet 4, creating transactions version 2 by default, and supporting single-descriptor wallets!
+- **The Book of BDK has seen significant improvements.** After a few months of use and feedback on the book, we have updated a ton of pages and examples to reflect some of the new and unclear questions we were getting from users on our Discord server and on GitHub. Check out [https://bookofbdk.com](https://bookofbdk.com)!
+- **Kyoto (a new Compact Block Filters client) is now usable with BDK.** The [bdk-kyoto](https://github.com/bitcoindevkit/bdk-kyoto) library moved into the [Bitcoin Dev Kit GitHub organization](), and while the client should be considered experimental, it can now be used with directly with BDK wallets!
```

Comment:
```suggestion
- **Kyoto (a new Compact Block Filters client) is now usable with BDK.** The [bdk-kyoto](https://github.com/bitcoindevkit/bdk-kyoto) library moved into the [Bitcoin Dev Kit GitHub organization](), and while the client should be considered experimental, it can now be used directly with BDK wallets!
```

Reply:
Good catch!